src/helpers/safeGetPostsAmountFromContract.ts

```diff
@@ -0,0 +1,10 @@
+import { SCPostStorage } from '@big-whale-labs/seal-cred-posts-contract'
+
+export default async function (contract: SCPostStorage) {
+  try {
+    return Number(await contract.currentPostId())
+  } catch (error) {
+    console.error('Error occurred while running "currentPostId()"', error)
```

Comment:
Let's send it into `handleError('Error occurred while getting current post id from the contract', error)`, note the message changes, I think it sounds better

Reply:
Also, looks like you have nested `try-catch` blocks, not sure if should do this